examples/cosmic_dance.jl

```diff
@@ -0,0 +1,58 @@
+using Javis
+
+function object(r, p = O, color = "black")
+    sethue(color)
+    circle(p, r, :fill)
+    return p
+end
+
+function connector!(connection, p1, p2, color)
+    sethue(color)
+    push!(connection, [p1, p2])
+    map(x -> line(x[1], x[2], :stroke), connection)
+end
+
+function ground(args...)
+    background("black") # canvas background
+    sethue("white") # pen color
+    # circle(O,50,:fill)
+end
+
+function circ(p = O, color = "black", action = :fill, radius = 25, edge = "solid")
+    sethue(color)
+    setdash(edge)
+    circle(p, radius, action)
+end
+
+function make_animation()
+
+    # to store the connectors
+    connection = []
```

Comment:
Maybe make this `Point[]`

Reply:
I am confused...I wanted it to store something like this `(point where the line begins, point where the line ends)` so that the function `connector!` that I wrote will work

Reply:
Oh sorry it could be `Vector{Tuple{Point, Point}}` I guess and make the `push!` as a tuple. But it's not that important for this example :smile:

Reply:
ohh yeah I did not think of a `tuple` :sweat_smile:

Reply:
Did you wanna change it to it? Otherwise we'll merge this soon

Reply:
I guess I will go with the code that is there i.e., without changing.